README.md

```diff
@@ -86,6 +86,7 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [Entrust](https://entrust.wd1.myworkdayjobs.com/EntrustCareers/job/United-States---Sunrise-FL/Software-Engineering-Intern_R002369) | Sunrise, FL | Software Engineer Intern |
 | [Enphase Energy](https://jobs.jobvite.com/enphase-energy/job/oFPRmfwr) | Austin, TX | Embedded Firmware Engineer Intern |
 | [Körber](https://jobs.koerber.com/pharma/job/Fargo-Software-Developer-Intern-ND-58102/954269201/) | Fargo, ND | Software Developer Intern |
+| [Tokio](https://tokio.uk.ms/careers) | Remote | Software Developer Intern |
```

Comment:
This link looks suspicious to me. Tokio's official site is https://tokio.rs, not https://tokio.uk.ms, and I can't confirm whether the latter is an official, alternative domain. Furthermore, there's no equivalent careers site on the official .rs site. In fact, the .uk.ms site is a perfect clone of the official .rs site _except_ that the Blog tab is replaced with a Careers tab. For security reasons, please change the link to a careers page on the official Tokio site instead.

Reply:
Looks like you already took the initiative to ask on Tokio's official GitHub page. 

The fact that this page looks identical to the official page makes me skeptical that this is an official Tokio website. Also, I feel like if this was real, they would put more effort into ensuring that their application system works — right now, you could put nothing in and press submit, and it would blindly accept it. 

I wouldn't merge.

Reply:
TIL why I should wait until an application is merged to apply

Reply:
@kevinroosey TBF, it's possible that we maintainers accidentally let some suspicious postings through from time to time, so I definitely encourage taking the initiative to verify postings independently (and please open a PR if you find something!)